### PR TITLE
progress in pop up html / css

### PR DESCRIPTION
**Pull request by harae37**

### CODE REVIEW

File: addon/popup/index.html [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+  <title>popup</title>
   <link rel="stylesheet" href="css/styles.css"/>
 </head>
+
+
 <body>
+  <div id="popup-content">
+    <div class="title"> DATA BODIES </div>
+    <div class="logo">  </div>
+    <div class="button"> <a href="../../www/index.html"> Visit </a> </div>
```
Comment (nbriz) [inline addon/popup/index.html]: i can see why u would write the path to the website this way, but keep in mind that the addon && the website will exist on different computers (the website is hosted on a server somewhere... we haven't set that up yet but will eventually) && the addon runs on the computer of the individual visitor (ie. on someone's browser), so we can't write relative paths like this. it needs to be an absolute path or more specifically a web-address, something like: http://ourwebsite.com (i imagine u all haven't purchased a domain name yet, so u might want to talk about what that should be as a group), once u've got one purchased u can replace this path w/that address but for now let's change this to "http://WEBSITE-NAME-GOES-HERE.com" so we don't forget

**Review by nbriz — CHANGES_REQUESTED** [review]
have u tested this addon in ur browser? i tried to load it into firefox but it did not work.

for reference the way u load it in firefox is:
1. visit `about:debugging` (in ur address bar) 
2. then on the left u click on "This Firefox"
3. then u click the "Load Temporary Add-on" button. once u do that u can choose the "manifest.json" file && the addon should load. 

however, in this case the addon won't load && the issue seems to be w/the manifest.json file. i know that u didn't create the initial file (i believe that was @Steve-luo ), but i think u should try to fix it before i accept this pull request. 

the first issue i noticed is that there is a missing comma here:  https://github.com/harae37/data-bodies/blob/f7408300d2be5b6702e74ac0628ad8704d68ac4b/addon/manifest.json#L12

once i added that comma && tried to load the addon i got an error saying that said:
```
Extension is invalid
Reading manifest: Error processing content_scripts.0: Property "matches" is required
```
take a look at the [documentation for the manifest.json](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json) file && see if u can figure out what/where code is missing.

**Comment by harae37** [comment]
Got it. I changed the link to the website into the temporary address. But I'm still unsure of the js part, bc I didn't put in manifest.json into the html yet. Which part should I put it in?

**Comment by nbriz** [comment]
@harae37 the `addon/manifest.json` file is a "meta data" file. u don't need to include it in any HTML page or anything like that. instead, it's the browser that uses this file when u try to load a new addon. see my instructions for how to load an addon in firefox in my previous comment.

**Comment by harae37** [comment]
For adding the missing comma in line 14 should it be "browser_action":, or "browser_action",: ?

**Comment by nbriz** [comment]
it's not line 14 that's missing the comma it's line 12:  https://github.com/harae37/data-bodies/blob/f7408300d2be5b6702e74ac0628ad8704d68ac4b/addon/manifest.json#L12

take a look at the rest of the file && notice where the commas are, u'll notice that line 12 should have one (based on the rest of the file's syntax)

**Comment by harae37** [comment]
Got it. I added the comma and tried to test on Firefox, but now it's telling me :

" JSON.parse: expected double-quoted property name at line 12 column 3 of the JSON data". 

I don't think I'm seeing any missing quote marks, do you see them by any chance?

**Comment by nbriz** [comment]
it looks like u also added commas to the last items in the arrays. this is known as a "trailing comma", for example here's a javascript array with no trailing comma:
```js
x = [  "nick", "tina", "andy"  ]
```
now here it is again but this time with a trailing comma (after "andy")
```js
x = [  "nick", "tina", "andy",  ]
```
in JavaScript this is ok (b/c as we discussed in class JavaScript is very good at fixing our tiny mistakes), but in JSON (which is a data format inspired by JavaScript, but it's not actually JavaScript) the syntax is much more strict. so if u add trailing commas (ie commas after the last item in an array) it won't work


also, i noticed u added the "matches" property (that's great!) but u have it set to "*://*.mozilla.org/*" this mean the addon will only run on pages that "match" that pattern, ie. only on mozilla.org pages. i'm guessing u want it to run on every website u visit (not just mozilla's page) so u'll want to do some research && see what the correct setting for matches is in this case.

**Comment by harae37** [comment]
I changed the mozilla.org into http://*/*, and tested and I think it worked (it didn't give me the error box).

**Comment by nbriz** [comment]
looks good, i tried it too && it worked on my end :)
i'll accept the pull request

**Pull request merged**